liblocation/src/main/java/com/mapbox/android/core/location/LocationEngine.java

```diff
@@ -130,7 +130,7 @@ public int getInterval() {
    * Set the rate in milliseconds at which your app prefers to receive location updates. Note that the location updates
    * may be faster than this rate if another app is receiving updates at a faster rate, or slower than this rate, or
    * there may be no updates at all (if the device has no connectivity, for example). This has no effect on
-   * AndroidLocationEngine, as it uses fastestInterval to set minTime.
+   * <tt>AndroidLocationEngine</tt>, as it uses fastestInterval to set minTime.
```

Comment:
It'd be great to also format `fastestInterval` and `minTime` the same way because those refer to code variables.